### DISCUSSION TITLE
Wyze Cam V3 rebooting every 5-10 minutes

### DISCUSSION THREAD

**Discussion by HeyItsDaddy** [gtxaspec/wz_mini_hacks#803] [Q&A]
I redid all 6 of my cameras yesterday, made them virtually identical with the exception of their names have a 1 through 6 in them.  My first camera, though, is rebooting every couple of minutes (not at an exact interval, it's random, but short).  I've looked through every log I can think of, pulled the SD card and run diagnostics on it, switched out the power supply with a different one, etc.  I'm at a loss of what to try next.

Is there any way to see any type of event logging or try to track down what was happening when it is rebooting?  All of the Linux commands/procedures I've looked up don't work, of course, because of the scaled down nature of the OS/firmware.  Any other thoughts or ideas would be greatly appreciated.

**Comment by HeyItsDaddy** [comment]
Re-flashed firmware, and formatted SD card and copied fresh wz_mini_hacks.  Under 10 minutes later...reboot.  :-|
Unpackaged a new 32GB SD card and used it instead...7 minutes later, reboot.  *sigh*
The other 5 cameras aren't having this issue.

**Comment by HeyItsDaddy** [comment]
OK, I was wrong...it IS happening on my other cameras, too.  Going to try flipping the DISABLE_FW_UPGRADE back to false for an evening and see if it resolves.  If so, then there must be some kind of check or what not going on that we haven't accounted for.  I'll let you know tomorrow if that resolves it.

**Comment by HeyItsDaddy** [comment]
Checked, and that did fix the issue...so the workaround to spoof the FW version is making the cameras crash constantly.  Ideas, @gtxaspec ?

**Comment by dudududodododedede** [comment]
Yep I'm also seeing this issue on my V3 cams on the old firmware 4.36.9.139

**Comment by tismofied** [comment]
Same here. I reverted the DISABLE_FW_UPGRADE to false and no issues. As a matter of fact, It also breaks the poe aspect since my UCTRONICS never adopts an ip with the firmware disable variable set to true.

**Comment by 1git** [comment]
+1 Random reboots every 5-20 mins when DISABLE_FW_UPGRADE = true 
everything works when set to false

**Comment by HeyItsDaddy** [comment]
It's one of the more useful mini hacks since Wyze started forcing the latest firmware updates to even use the app...but not so great when they reboot constantly.  :-(  If you have any insight, @gtxaspec , please let us know.  Keep up the great work!

**Comment by 1git** [comment]
So an interesting find...whilst poking about... there is a hidden .ota folder in the /media/mmc/ directory...
comparing files form a working (not rebooting) and the broken v3 I found one file different...

```
ls -lah /media/mmc/.ota/
-rwxrwxrwx    1 root     root        1.3M Jan 17 19:50 278a1887
-rwxrwxrwx    1 root     root           2 Dec  5 04:33 303a468b
-rwxrwxrwx    1 root     root           9 Jan 17 19:50 55c2509d
-rwxrwxrwx    1 root     root      609.4K Dec  5 02:55 58f8b10c
-rwxrwxrwx    1 root     root      372.1K Dec  5 02:55 75a23fe0
-rwxrwxrwx    1 root     root         557 Jan 17 19:50 779b258a
-rwxrwxrwx    1 root     root         465 Dec  5 04:38 79cf8e35
-rwxrwxrwx    1 root     root     1018.0K Jan 17 19:50 88b5bc5f
-rwxrwxrwx    1 root     root         610 Dec  5 02:55 ac805811
-rwxrwxrwx    1 root     root         640 Dec  5 04:38 b2ea655d
-rwxrwxrwx    1 root     root        6.4K Dec  5 04:33 cc4b26d3
-rwxrwxrwx    1 root     root          39 Dec  5 02:55 d8cc3d85
-rwxrwxrwx    1 root     root      340.1K Dec  5 04:33 e526e333
-rwxrwxrwx    1 root     root       84.9K Dec  5 04:34 fc723ec0
```

779b258a was a smaller size `398 Jan 25 23:57 779b258a` on the camera that was rebooting all the time...
I copied it across and re-enabled the DISABLE_FW_UPGRADE = true...

so far:
```
uptime
 07:23:31 up 21 min,  0 users,  load average: 2.68, 2.77, 1.93
```

I'll leave it running over night and report back in the morning.

**Reply by gtxaspec** [reply]
the .ota folder contains a split archive of programs that are downloaded over the air from wyze, like the webrtc streaming stuff for the web interface.

**Reply by 1git** [reply]
ok so I just checked it and 
```
uptime
 14:44:21 up 17 min,  0 users,  load average: 3.37, 2.58, 1.79
```
So That didn't fix it...

:(

**Reply by 1git** [reply]
So I even tried cloning a working sd card (using DD) to a new SD card, minor changes in the wz_mini.conf file (name) 
and after a few hours it crashed again, when the DISABLE_FW_UPGRADE="true"
So this seems to be something to do with the camera?
both are flashed to demo_wcv3_4.36.9.131

Any ideas?

@gtxaspec

**Reply by 1git** [reply]
From logread:
```
Jan 26 21:16:06 iCamera: Segmentation fault (core dumped)
Jan 26 21:16:08 sinker: [EdgeAI Process ERROR]-[mngSock.cpp:219:sockHandle]fault to call connect(No such file or directory)
Jan 26 21:16:08 sinker: [EdgeAI Process INFO]-[imageHandle.cpp:187:sockHandle]exit sock (3)
Jan 26 21:16:08 assis: [/system/bin/sysMonitor.sh] send sig to assis!
Jan 26 21:16:08 syslog: assisprocess.c,37:########################
Jan 26 21:16:08 syslog: assisprocess.c,38:assis capture signal: 16
Jan 26 21:16:08 syslog: assisprocess.c,39:########################
Jan 26 21:16:08 audiocardprocess: [watchdog]dbg: soundcard_watchdog_snd_msg_feed [time:1737951368]
Jan 26 21:16:15 sinker: [EdgeAI Process ERROR]-[mngSock.cpp:219:sockHandle]fault to call connect(No such file or directory)
Jan 26 21:16:15 sinker: [EdgeAI Process INFO]-[imageHandle.cpp:187:sockHandle]exit sock (3)
Jan 26 21:16:22 sinker: [EdgeAI Process ERROR]-[mngSock.cpp:219:sockHandle]fault to call connect(No such file or directory)
Jan 26 21:16:22 sinker: [EdgeAI Process INFO]-[imageHandle.cpp:187:sockHandle]exit sock (3)
Jan 26 21:16:24 audiocardprocess: [watchdog]dbg: soundcard_watchdog_snd_msg_feed [time:1737951384]
Jan 26 21:16:28 syslog: log/resetlog.c,48:gmtoff = -25200 t = 21
Jan 26 21:16:28 syslog: log/resetlog.c,60:acDesPath = /media/mmc/rebootlog/reboot_D03F278E47D4_20250126211628_s.txt
Jan 26 21:16:28 syslog: [====== [log_info_snap] ======]: 
Jan 26 21:16:29 sinker: [EdgeAI Process ERROR]-[mngSock.cpp:219:sockHandle]fault to call connect(No such file or directory)
Jan 26 21:16:29 sinker: [EdgeAI Process INFO]-[imageHandle.cpp:187:sockHandle]exit sock (3)
```

from dmesg:
```
Jan 26 21:16:30 syslog: [  327.892400] do_page_fault() #2: sending SIGSEGV to alarm-record for invalid read access from
Jan 26 21:16:30 syslog: [  327.892400] 4e6b5004 (epc == 7694dbd0, ra == 7694dbcc)
Jan 26 21:16:30 syslog: [  327.941456] SPEAKER CTL MODE0 !
Jan 26 21:16:30 syslog: [  328.094932] gc2053 stream off
Jan 26 21:16:30 syslog: [  328.267056] codec_codec_ctl: set CODEC_TURN_OFF...
Jan 26 21:16:30 syslog: [  328.444993] codec_codec_ctl: set CODEC_TURN_OFF...
```
and:

```
Jan 26 21:16:31 syslog: log/assistlog.c,305:cmd:logtool -e /tmp/log.package -f /media/mmc/rebootlog/reboot_D03F278E47D4_20250126211628_s.txt
Jan 26 21:16:31 assis: encryption has been done
Jan 26 21:16:37 sinker: [EdgeAI Process ERROR]-[mngSock.cpp:219:sockHandle]fault to call connect(No such file or directory)
Jan 26 21:16:37 sinker: [EdgeAI Process INFO]-[imageHandle.cpp:187:sockHandle]exit sock (3)
Jan 26 21:16:39 audiocardprocess: [watchdog]dbg: soundcard_watchdog_snd_msg_feed [time:1737951399]
Jan 26 21:16:44 sinker: [EdgeAI Process ERROR]-[mngSock.cpp:219:sockHandle]fault to call connect(No such file or directory)
Jan 26 21:16:44 sinker: [EdgeAI Process INFO]-[imageHandle.cpp:187:sockHandle]exit sock (3)
Jan 26 21:16:51 sinker: [EdgeAI Process ERROR]-[mngSock.cpp:219:sockHandle]fault to call connect(No such file or directory)
Jan 26 21:16:51 sinker: [EdgeAI Process INFO]-[imageHandle.cpp:187:sockHandle]exit sock (3)
```

**Reply by jaajee** [reply]
> So an interesting find...whilst poking about... there is a hidden .ota folder in the /media/mmc/ directory... comparing files form a working (not rebooting) and the broken v3 I found one file different...
> 
> ```
> ls -lah /media/mmc/.ota/
> -rwxrwxrwx    1 root     root        1.3M Jan 17 19:50 278a1887
> -rwxrwxrwx    1 root     root           2 Dec  5 04:33 303a468b
> -rwxrwxrwx    1 root     root           9 Jan 17 19:50 55c2509d
> -rwxrwxrwx    1 root     root      609.4K Dec  5 02:55 58f8b10c
> -rwxrwxrwx    1 root     root      372.1K Dec  5 02:55 75a23fe0
> -rwxrwxrwx    1 root     root         557 Jan 17 19:50 779b258a
> -rwxrwxrwx    1 root     root         465 Dec  5 04:38 79cf8e35
> -rwxrwxrwx    1 root     root     1018.0K Jan 17 19:50 88b5bc5f
> -rwxrwxrwx    1 root     root         610 Dec  5 02:55 ac805811
> -rwxrwxrwx    1 root     root         640 Dec  5 04:38 b2ea655d
> -rwxrwxrwx    1 root     root        6.4K Dec  5 04:33 cc4b26d3
> -rwxrwxrwx    1 root     root          39 Dec  5 02:55 d8cc3d85
> -rwxrwxrwx    1 root     root      340.1K Dec  5 04:33 e526e333
> -rwxrwxrwx    1 root     root       84.9K Dec  5 04:34 fc723ec0
> ```
> 
> 779b258a was a smaller size `398 Jan 25 23:57 779b258a` on the camera that was rebooting all the time... I copied it across and re-enabled the DISABLE_FW_UPGRADE = true...
> 
> so far:
> 
> ```
> uptime
>  07:23:31 up 21 min,  0 users,  load average: 2.68, 2.77, 1.93
> ```
> 
> I'll leave it running over night and report back in the morning.

I have other way around : 
```
-rwxrwxrwx    1 root     root      743.1K Feb  1 14:06 1e26fd4b
-rwxrwxrwx    1 root     root        1.3M Jan 10 20:02 278a1887
-rwxrwxrwx    1 root     root           2 Dec 29 17:15 303a468b
-rwxrwxrwx    1 root     root           9 Jan 10 20:02 55c2509d
-rwxrwxrwx    1 root     root      609.4K Dec 29 17:15 58f8b10c
-rwxrwxrwx    1 root     root      372.1K Dec 29 17:26 75a23fe0
-rwxrwxrwx    1 root     root         557 Jan 10 20:02 779b258a
-rwxrwxrwx    1 root     root         465 Dec 31 15:57 79cf8e35
-rwxrwxrwx    1 root     root     1018.0K Jan 10 20:02 88b5bc5f
-rwxrwxrwx    1 root     root         610 Dec 29 17:26 ac805811
-rwxrwxrwx    1 root     root         640 Dec 31 15:57 b2ea655d
-rwxrwxrwx    1 root     root         308 Dec 31 15:57 bb957b8a
-rwxrwxrwx    1 root     root        6.4K Dec 29 17:15 cc4b26d3
-rwxrwxrwx    1 root     root          39 Dec 29 17:26 d8cc3d85
-rwxrwxrwx    1 root     root      340.1K Dec 29 17:15 e526e333
-rwxrwxrwx    1 root     root       84.9K Dec 29 17:15 fc723ec0
```

I have three cameras with wcv3_4.36.9.131 setup and all have DISABLE_FW_UPGRADE="true". Interestingly, one camera which doesn't reboot often, does not have `-rwxrwxrwx    1 root     root      743.1K Feb  1 14:06 1e26fd4b` in .ota folder. Other two cameras do have this file and they reboot in every 10 -15 minutes.
I tried to delete `1e26fd4b`, but after reboot it comes back. Don't know what is causing this issue.

**Comment by patchido** [comment]
I am having the same issue, anybody got a workaround? is it not happening to everyone? maybe th eonly workaround is to deny the connection to wyze app 100% and just use the local rtsp?

**Reply by tismofied** [reply]
Hey @patchido have a look at thingino project. It's a very polished firmware being developed by gtxaspec himself.

**Comment by sideup66** [comment]
Late to the party on this chat. What is likely going on here is the script is working as intended. Wyze a while back now announced a min app version, i suspect this is due to some change or security improvement on their side that breaks compatibility with the old versions below a certain level. Its possible some way they are detecting the devices are still on old firmware and are now simply trying to force them over to new.

I DID actually move my camera to the dark side for my camv3 and found that version 4.36.11.7095 is stable for me, and seems to work fine with wz_mini_hacks. Only side effect seems to be the record light now flashes if motion detection is enabled and captures motion outside the camera cooldown. I dont consider that necessarily an issue as it now just simply  indicates when it is recording vs not in both local and remote sense. I would consider flashing like this: 1. remove the sd card from your camera and boot without it. 2. use a spare sd card preferrebly to flash the firmware version over. once booted and ok shutdown and re install your normal sd with wz_mini.

**Comment by 1git** [comment]
I seem to have found some progress on this one... still testing... but... sharing here anyway....
I refreshed my V3's with https://download.wyzecam.com/firmware/v3/demo_wcv3_4.36.9.131.zip
and enabled the the DISABLE_FW_UPGRADE (= true) and it seems to be stable so far... 3 days in...

**Reply by keeepinitgansta** [reply]
hmm I will give this FW a shot to see if it's more stable.

Edit: No joy, still the same behavior on 4.36.9.131

EDIT2: Downgraded to FW version 4.36.7.22 and that solved it for me.

**Comment by HeyItsDaddy** [comment]
So, @1git , you wheeled it back from 4.36.9.139 to 4.36.9.131, and found it stops the reboot loops?  How is that going?
And @sideup66 you found that the wz_mini_hacks DO work with 4.36.11.7095?  (The one just prior to the "app and firmware security improvements" Wyze did that prevents them.)  How is that going?  I'm about to redo/refresh my cams again, which takes a while, so would be nice to know which direction to go.

**Comment by sideup66** [comment]
@HeyItsDaddy I am still on 4.36.11.7095. Aside from a glitchy LED when it is in some states and recording (it flashes blue and red and flickers a bit) my camera seems to work fine. I've paired that on android with app version 2.50.7.487...can find that on apkmirror and functionally mine seems to work fine. 
Mind you the rtsp stuff doesnt work on this version but if you are a hybrid setup (hacks to make the thing not suck + stock stuff) it actually makes the camera run pretty smooth. Id suggest that build if you want the dark side. Disable automatic updates and the firmware spoof however if you want that. I have a PR also for @gtxaspec to integrate that adds an option to disable that.

**Comment by endertable** [comment]
@sideup66 do you even need the firmware spoof for version 7095?  I do have an older version of the app on my iPhone 2.50.6 although it is starting to bark at me about updating to 3.8.5.11.  My version does not give me any problem with 7095 nor the prior 2 versions but it did not allow access to the popular hackable 139.  I use WebRTC to view my cams and for commands I slapped together almost like a mini-fied wyze-docker-bridge, without all the video and docker and bridge stuff.
